apps/dashboard/src/pages/edit-workflow.tsx

```diff
@@ -10,7 +10,7 @@ export const EditWorkflowPage = () => {
       <EditWorkflowLayout headerStartItems={<EditorBreadcrumbs />}>
         <div className="flex h-full flex-1 flex-nowrap">
           <WorkflowTabs />
-          <aside className="text-foreground-950 [&_textarea]:text-neutral-600'; flex h-full w-[300px] max-w-[350px] flex-col border-l [&_input]:text-xs [&_input]:text-neutral-600 [&_label]:text-xs [&_label]:font-medium [&_textarea]:text-xs">
+          <aside className="text-foreground-950 [&_textarea]:text-neutral-600'; flex h-full w-[350px] max-w-[350px] flex-col border-l [&_input]:text-xs [&_input]:text-neutral-600 [&_label]:text-xs [&_label]:font-medium [&_textarea]:text-xs">
```

Comment:
Unfortunately, I had to make the sidebar wider, otherwise the scheduled digest UI doesn't fit and look well.